src/internal/module/module.go

```diff
@@ -16,7 +21,32 @@ type Module struct {
 	TargetSystem string // The module target system
 }
 
+func (m Module) Logger(logger *slog.Logger) *slog.Logger {
+	return logger.With(slog.String("namespace", m.Namespace), slog.String("name", m.Name), slog.String("targetSystem", m.TargetSystem))
+}
+
 func (m Module) RepositoryURL() string {
-	repoName := fmt.Sprintf("terraform-%s-%s", m.TargetSystem, m.Name)
-	return fmt.Sprintf("https://github.com/%s/%s", m.Namespace, repoName)
+	return fmt.Sprintf("https://github.com/%s/terraform-%s-%s", m.Namespace, m.TargetSystem, m.Name)
```

Comment:
We should be careful here, if we ever want to introduce a proxy or support anything else we may not want to hardcode github.com here.

What I would recommend is that we have a method on the github client where you can pass it an org/repo and it will generate you the url.